Library/Homebrew/cmd/cat.rb

```diff
@@ -3,8 +3,9 @@ def cat
     # do not "fix" this to support multiple arguments, the output would be
     # unparsable, if the user wants to cat multiple formula they can call
     # brew cat multiple times.
+    formulae = ARGV.formulae
+    raise FormulaUnspecifiedError if formulae.empty?
```

Comment:
Maybe it's a good idea to add `raise "brew cat doesn't support multiple arguments" if formulae.size > 1`